src/trajectory/Interpolated.cpp

```diff
@@ -90,11 +90,11 @@ void Interpolated::evaluate(double _t, State *_state) const
 void Interpolated::evaluateDerivative(double _t, int _derivative,
   Eigen::VectorXd& _tangentVector ) const
 {
-  if (_derivative == 0)
+  if (_derivative <= 0) // Invalid
```

Comment:
I didn't realize I pushed this edit as well. This has nothing to do with warnings. Recommend I remove it?